src/test/java/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition2Test.java

```diff
@@ -634,4 +634,72 @@ public void scriptInitializerCallsCpsTransformedMethod() throws Exception {
         jenkins.assertLogContains("CpsCallableInvocation{methodName=bar,", b);
     }
 
+    @Issue("SECURITY-1465")
+    @Test public void blockLhsInMethodPointerExpression() throws Exception {
+        WorkflowJob p = jenkins.jenkins.createProject(WorkflowJob.class, "p");
```

Comment:
This suite is a really good candidate for `@ClassRule` BTW. Maybe pay that tax now, so you are decreasing rather than increasing test time? (And stop using `AbstractCpsFlowTest`. Its utilities are not used here AFAIK, and it ought to be deprecated or deleted.)

Reply:
Fair enough, I'll look into using `@ClassRule` here.

Reply:
Done in 40b09db and 5127785.